Bugzilla/Config.pm

```diff
@@ -307,25 +307,26 @@ sub write_params {
         my $ok = eval {
             $dbh->do("DELETE FROM params");
             my $sth = $dbh->prepare("INSERT INTO params (param_key, param_val) VALUES (?, ?)");
+            trick_taint($params->{$_}) if defined $params->{$_};
             $sth->execute($_, $params->{$_}) foreach keys %$params;
             $dbh->bz_commit_transaction();
             1;
         };
```

Comment:
This does not work. Gives taint error still. Needs to be instead:

     my $ok = eval {
        $dbh->do("DELETE FROM params");
        my $sth = $dbh->prepare("INSERT INTO params (param_key, param_val) VALUES (?, ?)");
        foreach my $key (keys %$params) {
          trick_taint($params->{$key}) if defined $params->{$key};
          $sth->execute($key, $params->{$key});
      }
      $dbh->bz_commit_transaction();
      1;
   };